connectors/vault/Makefile

```diff
@@ -10,7 +10,7 @@ VAULT_PATH=${ABSTOOLBIN}/vault
 include $(ROOT_DIR)/pkg/policy-compiler/policy-compiler.env
 
 docker-build:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o vault-connector vault_connector.go vault_utils.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o vault-connector vault_connector.go
```

Comment:
Please add the ` go test -v ./...` also in the test target in line 50 in this Makefile. Otherwise the CI won't run the test that you just added.

Reply:
fixed. Thanks.